LICENSE.md

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2018-2022 Anthony De Meulemeester (@anthdm), City of Zion
+community (@CityOfZion), NeoSPCC (@nspcc-dev)
```

Comment:
Not sure whether we should include Anthony De Meulemeester and City of Zion community, they are not in the GitHub repo contributors list.

Reply:
Not in this repository, Go dBFT library is written by NeoSPCC entirely.

Reply:
BTW, maybe _multiline_ copyright is what makes GH freak out on our NeoGo license? I remember it worked once upon a time.